src/resources/js/__mocks__/jquery.js

```diff
@@ -0,0 +1,28 @@
+const jQuery = require('jquery/dist/jquery.js');
+const { parse } = require('graphql');
+import product from '../jest_mock_objects/product';
+import products from '../jest_mock_objects/products';
+import cartProducts from '../jest_mock_objects/cartProducts';
+
+jQuery.ajax = settings => {
```

Comment:
@AlexeiaW one thing we're not testing right now is validity of the graphql query. Because we're automatically returning a mocked response, the graphql query is always seen to be "correct". So we can introduce a graphql mock server and compare our mocked object from graphql (properties rather than values), with our mocked js object. 

Have a look here:

https://graphql.org/blog/mocking-with-graphql/